### PR TITLE
fix: various typos

### PR DESCRIPTION
**Pull request by nschonni**
## Summary

Another pass with cSpell

### CODE REVIEW

File: docs/framework/data/adonet/connecting-to-a-data-source.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 description: "Learn more about: Connecting to a Data Source in ADO.NET"
 title: "Connecting to a Data Source"
-deescription: Learn about Connection objects, used to connect to data sources in ADO.NET. The Connection object you choose depends on the type of data source.
+description: Learn about Connection objects, used to connect to data sources in ADO.NET. The Connection object you choose depends on the type of data source.
```
Comment (nschonni) [inline docs/framework/data/adonet/connecting-to-a-data-source.md:4]: Getting a warning since there is already a key, but not sure which one to keep

Comment (BillWagner) [reply docs/framework/data/adonet/connecting-to-a-data-source.md:4]: Thanks. The longer description on line 4 should be kept.

Comment (nschonni) [reply docs/framework/data/adonet/connecting-to-a-data-source.md:3]: OK, deleted the first one

**Review by BillWagner — APPROVED** [review]
I gave everything a reasonable look, and this is good to merge.

I'll work around why the C++ projects aren't building. That's not related to this PR.

**Comment by BillWagner** [comment]
closing and reopening for a fresh build.

**Pull request merged**